content/en/contributing/multi-account-region-testing.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Multi account and region testing"
+weight: 7
+description: >
+  How to test multi-account and multi-region compatibility.
+---
+
+LocalStack supports multi accounts and regions compatibility. This document describes how to test your changes against cross accounts and regions.
```

Comment:
Please prefer having one sentence on each line, this helps with diffs in future changes.